packages/botonic-nlp/src/storage/model-storage.ts

```diff
@@ -0,0 +1,13 @@
+import { LayersModel, loadLayersModel } from '@tensorflow/tfjs-node'
+import { mkdirSync } from 'fs'
+
+export class ModelStorage {
+  static async load(path: string): Promise<LayersModel> {
+    return await loadLayersModel(`file://${path}/model.json`)
```

Comment:
add model.json constant in storage/constants.json?

Reply:
agree